src/index.js

```diff
@@ -1,8 +1,8 @@
 import './style.css';
 import Icon from './option.png';
-/* eslint-disable-next-line import/no-cycle */
 import validateForm from './funct.js';
 import removeTask from './remove.js';
+import removeAllTask from './removeAll.js';
```

Comment:
- [Optional] I noticed that all the required features work, however, I could delete the list by clicking the `ellipse` icon which is not intuitive to the users, as they will expect a delete icon, Please add a delete icon clickable to trigger the delete function.

[Screencast from 05-17-2023 04:20:31 PM.webm](https://github.com/MenendezON/WP-To-Do-list/assets/58771507/58d88b09-0e13-4b87-b7b5-05c38c2d29c8)